internal/testrunner/coverageoutput.go

```diff
@@ -42,11 +43,17 @@ func (tcd *testCoverageDetails) withTestResults(results []TestResult) *testCover
 			tcd.dataStreams[result.DataStream] = []string{}
 		}
 		tcd.dataStreams[result.DataStream] = append(tcd.dataStreams[result.DataStream], result.Name)
+		if tcd.cobertura != nil && result.Coverage != nil {
+			tcd.cobertura.Merge(result.Coverage)
+		} else {
+			tcd.cobertura = result.Coverage
```

Comment:
What happens if `result.Coverage` is nil and `tcd.cobertura` is not? In that line we will be setting the latter to nil. Is that correct?

Reply:
Fixed, thanks